$OpenBSD: patch-tests_test_utils_py,v 1.1 2019/06/29 21:42:07 kmos Exp $

Fix tests for pytest 4. From a patch not committed upstream yet.

https://github.com/kennethreitz/requests/pull/5049

Index: tests/test_utils.py
--- tests/test_utils.py.orig
+++ tests/test_utils.py
@@ -33,7 +33,8 @@ class TestSuperLen:
         'stream, value', (
             (StringIO.StringIO, 'Test'),
             (BytesIO, b'Test'),
-            pytest.mark.skipif('cStringIO is None')((cStringIO, 'Test')),
+	    pytest.param(cStringIO, 'Test',
+            		 marks=pytest.mark.skipif('cStringIO is None')),
         ))
     def test_io_streams(self, stream, value):
         """Ensures that we properly deal with different kinds of IO streams."""
